firmware/mcal/stm32f767/os/tick.h

```diff
@@ -0,0 +1,26 @@
+/// @author Matteo Tullo
+/// @date 2024-02-24
+
+#pragma once
+
+#include <cstdint>
+
+#include "shared/util/os.h"
+#include "cmsis_os2.h"
+
+// TODO: Add comments and handle errors more robustly
+namespace os {
```

Comment:
namespace should be `mcal::os`

Reply:
The tick function will need to be bound to an app-level tick function in project/os.cc, it should not be directly implemented here as that requires the project to use the `os` namespace

Reply:
Will leave as is for now